vlei-verifier-workflows/src/utils/test-util.ts

```diff
@@ -627,83 +629,29 @@ export async function sendAdmitMessage(
   await markAndRemoveNotification(senderClient, grantNotification);
 }
 
-export async function getRootOfTrust(configJson: any): Promise<any> {
-  if (hasGLEIFWithMultisig(configJson)) {
-    return await getRootOfTrustMultisig(configJson);
-  } else {
-    return await getRootOfTrustSinglesig(configJson);
-  }
-}
-
-function hasGLEIFWithMultisig(data: any): boolean {
-  return data.users.some(
-    (user: any) =>
-      (user.type === "GLEIF" || user.type === "GLEIF_EXTERNAL") &&
-      user.identifiers.some((id: any) => data.identifiers[id]?.identifiers),
-  );
-}
+export async function getRootOfTrust(configJson: any, rot_aid: string, rot_member_aid?: string): Promise<any> {
```

Comment:
Can you add comments on this function so I can understand the implication of the rot_aid vs. the rot_member_aid?